sequgen/notebooks
# GGIR use case 1

Ref: sequgen issue [#21](https://github.com/sequgen/sequgen/issues/21)

## Description

Notebook for the first GGIR use case. Challenges:
1. use case requires boxcar function (implemented, see sequgen PR [#79](https://github.com/sequgen/sequgen/pull/79))
2. use case requires repeated parameter sampling (implemented with function `peaks_generate`)
3. use case requires relations between `Dimension` parameters (implemented with additional checks by functions `peaks_extends_maximum` and `peaks_overlap`)
4. use case does not require discrete time axis, adding this would remove information about the signal (implemented by not using the time axis and staying in the sample domain)

## To test

Install notebook and verify that it generates reasonable time series, compare with https://cran.r-project.org/web/packages/GGIR/vignettes/GGIR.html#42_Output_part_4 , tab 4.2.3 visualization_sleep.pdf
